### ISSUE TITLE
Story Line is over-populated

### ISSUE THREAD

**Issue by hugo-id** [olivierkes/manuskript#243]
Every plot and character, but no way to sort it, am I missing something?

Also there is no way to interact. A simple option to add resolution step for plots, or whatever equivalent for characters would be expected behaviour.

And I've opened like eight or ten issues, is there a way for me to leave positive feedback, or features I think are great?

**Comment by olivierkes** [comment]
Yes, Story Line is pretty much a work in progress. I'll have to get back to it. Thanks for your ideas!

-------

The "issue" mechanism is not the best way for positive feedbacks. You can leave positive feedbacks publicly, so that anyone can benefit:

- By writing documentation about it or tutoral, for example here in the wiki.
- By writing publicly about it: on a blog or social network or whatever.

If you don't like that, you can write me privately (I think you can through github, or through manuskript [website](www.theologeek.ch/manuskript)).  

:)
